packages/babel-preset-env/test/fixtures/debug/browserslists-last-2-versions-not-ie/options.json

```diff
@@ -0,0 +1,20 @@
+{
+  "validateLogs": true,
+  "ignoreOutput": true,
+  "presets": [
+    [
+      "env",
+      {
+        "debug": true,
+        "targets": {
+          "browsers": [
+            "last 2 versions",
+            "not ie <= 11",
+            "not ie_mob <= 11",
+            "not op_mob <= 12.1"
```

Comment:
This shouldn't be needed. It is a bug of browserslist that returns `op_mob 12.1` for `last 2 versions`, which should be instead `op_mob 59, op_mob 60`.